Lib/imaplib.py

```diff
@@ -902,6 +903,18 @@ def unsubscribe(self, mailbox):
         return self._simple_command('UNSUBSCRIBE', mailbox)
 
 
+    def unselect(self):
+        """Close the mailbox without expunging it.
```

Comment:
In the doc, in the docstring and in What's New in Python 3.9: you wrote 3 different documentation which are not the same. I would prefer to use the same documentation in the 3 places.

I'm not saying that your 3 definitions are wrong, but I'm now confused of which one is the most correct :-)